src/App.js

```diff
@@ -1,12 +1,70 @@
-import InputField from "./components/InputField";
+import React, { useState, useEffect } from "react";
+import Header from "./components/Header";
+import Input from "./components/Input";
+import NoMoviesFound from "./components/NoMoviesFound";
+import defaultMovieCards from "./defaultMovieCards";
+import Card from "./components/Card";
+import Cart from "./components/Cart";
+import getData from "./getData";
+import purchaseMovies from "./purchaseMovies";
 import "./App.css";
+import Loading from "./components/Loading";
+
+const App = () => {
+
+  const [userSearchInput, setUserSearchInput] = useState("");
+  const [data, setData] = useState(defaultMovieCards);
```

Comment:
We don't need to initiate dummy movies, we need to be an empty array as initial value and set the movies that you fetch when the app is up.